frodo.js

```diff
@@ -1,5 +1,5 @@
-/**
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.